config/locales/it.yml

```diff
@@ -20,6 +20,7 @@ it:
     has_many_new: "Aggiungi nuovo/a %{model}"
     has_many_delete: "Rimuovi"
     has_many_remove: "Rimuovi"
+    move: "sposta"
```

Comment:
You want this to start uppercase I guess?

Reply:
Capitalized I suppose. Done